src/calculateRentalCost.js

```diff
@@ -31,12 +31,12 @@ function calculateRentalCost(days) {
   // write code here
   let cost = 0;
 
-  if (days >= 3 && days < 7) {
+  if (days >= 0 && days <= 2) {
+    cost = days * 40;
```

Comment:
this action `days * 40` you can write once on the 32th line and use only a variable inside your IFs. If you use the same action or number twice this is a reason to reuse it  as a variable.

the cost without any discounts can be returned by default if any conditions are `false`. 

In addition, you can use only 2 `if` if in the first one you'll check the biggest number of days, in the second you'll check some mild numbers of days and if these two `if` will be false you return the default cost.